requirements.txt

```
@@ -11,3 +11,4 @@ pandas==1.3.5; python_version < "3.8"  # pandas 1.4 dropped python 3.7 support
 ruamel.yaml==0.17.24
 scipy==1.10.1; python_version >= "3.8"
 scipy==1.7.3; python_version < "3.8"   # scipy 1.8 dropped python 3.7 support
+linkml-runtime==1.5.0
```

```suggestion
linkml-runtime==1.5.0
```
Actually, never mind regarding my comments about the other requirements files. Sorry for the turnaround. I think it should not be in requirements.txt. Therefore, it should be in requirements-doc.txt. I'll add it back.

---

It needs to be in requirements for testing.